app/v2/controllers/V2DeparturesController.scala

```diff
@@ -237,4 +238,8 @@ class V2DeparturesControllerImpl @Inject() (
             response => Ok(Json.toJson(HateoasDepartureIdsResponse(response)))
           )
     }
+
+  def sendMessageDownstream(departureId: DepartureId): Action[Source[ByteString, _]] = Action(streamFromMemory) {
+    _ => Accepted(Json.obj("version" -> 2))
```

Comment:
Though it's just a stub and this will eventually be properly implemented, we need to handle the stream properly by doing something with it. We should do the following:

```suggestion
    implicit request => 
        // we must do something with the request otherwise it'll be held in memory.
        request.body.runWith(Sink.ignore)
        Accepted(Json.obj("version" -> 2))
```

Reply:
done